charts/nm-vllm-production-monitoring/templates/config-map-datasources.yaml

```diff
@@ -0,0 +1,23 @@
+{{- /*
+  A lot of inspiration taken from:
+    - https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/grafana/configmaps-datasources.yaml
+*/ -}}
+apiVersion: v1
+kind: ConfigMap
```

Comment:
I could never figure out of this applied to us, I assumed not since this config is so simple and has no secrets, but just will mention they recommend to make this type Secret: https://github.com/grafana/helm-charts/blob/main/charts/grafana/README.md#sidecar-for-datasources

Reply:
Interesting! I can see it applying in situations where credentials are involved, but there's nothing sensitive here, so a `ConfigMap` should be fine, I'd think.